locales/en/messages.json

```diff
@@ -70,6 +70,19 @@
         "message": "--- I can't find my device ---",
         "description": "Option in the port selection dropdown to allow the user to give permissions to the system to access the device."
     },
+    "portsSelectPermissionBluetooth": {
+        "message": "--- find my Bluetooth device ---",
```

Comment:
If we are going to add another element of this kind to the portpicker, maybe we must rename the old one:
`--- I can't find my device ---` -> `--- I can't find my USB device ---`
`--- find my Bluetooth device ---` -> `--- I can't find my Bluetooth device---`
Or similar, to have both texts similar and clear of what user can get at each one.
I know the USB in reality are SERIAL devices, but the user didn't know about that, they only know if the connection is made with an USB cable or with a Bluetooth dongle.
Another option is to have only one, that shows a modal window, asking wich kind of device to find (USB, Bluetooth, and we can add DFU), but this can be done in a future.

Reply:
Thanks. Updated.

Somehow port does not show up in portPicker. We might as well add an option like manual for bluetooth instead.